app/controllers/CheckYourAnswersController.scala

```diff
@@ -55,4 +65,22 @@ class CheckYourAnswersController @Inject()(
 
       Ok(view(sections))
   }
+
+  def onSubmit() = actions.authWithData.async {
+    implicit request =>
+
+      agentMapper.build(request.userAnswers) match {
+        case Some(agentDetails) =>
+          // TODO call estates store service with Agent Details and return to Estate Hub service
```

Comment:
How are you getting on with sending the information `agentDetails` to the estates backend?

Reply:
Updated